packages/apps-config/src/links/azeroId.ts

```diff
@@ -0,0 +1,71 @@
+// Copyright 2017-2023 @polkadot/apps-config authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import type { ApiPromise } from '@polkadot/api';
+import type { BN } from '@polkadot/util';
+import type { ExternalDef } from './types.js';
+
+import { resolveAddressToDomain, SupportedChainId } from '@azns/resolver-core';
+
+import { externalAzeroIdLogoBlackSVG, externalAzeroIdLogoPrimarySVG } from '../ui/logos/external/index.js';
```

Comment:
Redundant `/index.js`?

Reply:
I don't think it works without it 😢 it's some weird polkadot config quirk I guess